linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/ansible/shared.yml

```diff
@@ -1,3 +1,9 @@
+{{% if product in [ "sle12", "sle15" ] %}}
+{{% set aide_path = '/usr/bin/aide' %}}
+{{% else %}}
+{{% set aide_path = '/usr/sbin/aide' %}}
```

Comment:
To avoid defining this multiple times you should define it in products/sle15/product.yml, then add the deafult value to ssg/constants.py and handle the default value in ssg/products.py.

Reply:
ok. Will update the PR to do that.

Reply:
Done. I hope I did correctly what you've asked.